Removing DiactorosBootloader, adding Nyholm bridge instead (#708)
spiral/framework: CHANGELOG.md

  - Component `spiral/annotations` is removed. Use `spiral/attributes` instead.
  - Added return type `void` to a methods `publish`, `publishDirectory`, `ensureDirectory` in `Spiral\Module\PublisherInterface` interface.
  - Removed `Spiral\Http\SapiDispatcher` and `Spiral\Http\Emitter\SapiEmitter`. Please, use package `spiral/sapi-bridge` instead.

>   - Bootloader `Spiral\Bootloader\Http\DiactorosBootloader` is removed. Please, use package `spiral/nyholm-bridge` instead.

Reviewer: Could you add more info about NyholmBootloader here?

Author: Added

  - Classes `Spiral\Http\Diactoros\ResponseFactory`, `Spiral\Http\Diactoros\ServerRequestFactory`, `Spiral\Http\Diactoros\StreamFactory`,
    `Spiral\Http\Diactoros\UploadedFileFactory`, `Spiral\Http\Diactoros\UriFactory`

>     is removed. Please, use package `spiral/nyholm-bridge` instead.

Reviewer: ```suggestion
    are removed. You can use `spiral/nyholm-bridge`  to define PSR-17 factories.
```

Author: Fixed

  - Component `spiral/annotations` is removed. Use `spiral/attributes` instead.
  - Added return type `void` to a methods `publish`, `publishDirectory`, `ensureDirectory` in `Spiral\Module\PublisherInterface` interface.
  - Removed `Spiral\Http\SapiDispatcher` and `Spiral\Http\Emitter\SapiEmitter`. Please, use package `spiral/sapi-bridge` instead.

>   - Bootloader `Spiral\Bootloader\Http\DiactorosBootloader` is removed. Please, use package `spiral/nyholm-bridge` instead.
>     The bootloader `Spiral\Nyholm\Bootloader\NyholmBootloader` from the package `spiral/nyholm-bridge` registers the 
>     implementation of PSR-7/PSR-17 factories and completely replaces the functionality of the removed DiactorosBootloader.

Reviewer: ```suggestion
  - Bootloader `Spiral\Bootloader\Http\DiactorosBootloader` is removed. You can use the bootloader `Spiral\Nyholm\Bootloader\NyholmBootloader` from the package `spiral/nyholm-bridge` to register PSR-7/PSR-17 factories.
```

Mb it will be better
